packages/babel-helper-remap-async-to-generator/src/index.js

```diff
@@ -14,11 +14,24 @@ let buildWrapper = template(`
   })
 `);
 
+let arrowBuildWrapper =  template(`
+  (() => {
+    var ref = FUNCTION;
+    return (PARAMS) => ref.apply(this, arguments);
```

Comment:
Shouldn't this be a regular function expression and not an arrow function? (like in the default `buildWrapper`)